packages/vue-components/src/panels/PanelBase.js

```diff
@@ -80,12 +84,18 @@ export default {
     hasHeaderBool() {
       return this.$slots.header;
     },
+    expandHeaderlessBool() {
```

Comment:
hmm do we still need this?

Reply:
This is not needed anymore I will remove it, Thanks!